attendees_and_learners.rst

```diff
@@ -10,7 +10,7 @@ Workshops
 
 PyCon Ireland in Dublin, 26th October 2015
 ------------------------------------------
-
+Gearoid Ryan https://github.com/gearoid-ryan
```

Comment:
Please format as:

    * your name etc

(just make another change to your file)